api/src/routes/invites.ts

```diff
@@ -0,0 +1,82 @@
+import express from "express";
+import { isConstructorDeclaration } from "typescript";
+import Pod from "../../models/Pod";
+import PodInvites from "../../models/PodInvites";
+import User from "../../models/User";
+import auth, { AuthRequest } from "../authMiddleware";
+
+let invitesRouter = express.Router();
+
+invitesRouter.get(
+  "/currUsersInvites",
```

Comment:
I think we should be consistent between this route and GET /events - either by changing GET /events to GET /events/currUserEvents or by changing this route to just GET /invites